mautic/mautic
# [UX] Change added fields label when creating forms to be clearer (things like "Select" instead of "Radio group")

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [x]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
The current labels used for added fields during form creation, such as “Radio group”, could be made more intuitive. It would be advantageous to use more commonly understood terms. The proposed solution in this pull request is to change labels like “Radio group” to more universally understood terms like “Select”.

This change is beneficial for users who are in the process of creating forms and could benefit from more straightforward field labels. By implementing this change, we can expect a more user-friendly form creation process, potentially making the platform more accessible to a wider range of users.

**Before**:

- Select
- List - Country
- Text
- Textarea
- Description
- Checkbox group
- Radio group


**After**:

- Select
- Select: Country
- Text: Short answer
- Text: Paragraph
- Text: Article
- Checkbox: Multiple choices
- Checkbox: Single choice



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Forms page
3. Create a new form
4. Go to Fields tab
5. Click on Add a new field
6. Look at the list

![image](https://github.com/mautic/mautic/assets/116097999/266b7419-9c78-4306-8485-3920530a257d)
